core/src/components/tab-button/tab-button.ionic.scss

```
@@ -1,5 +1,7 @@
 @use "../../themes/ionic/ionic.globals.scss" as globals;
 @use "./tab-button.common";
+@import "../../themes/mixins";
```

As mentioned in a previous review, we don't use `@import` anymore because it's deprecated.

```suggestion
@use "../../themes/mixins";
```